src/xenia/gpu/command_processor.cc

```diff
@@ -1218,6 +1220,12 @@ bool CommandProcessor::ExecutePacketType3_DRAW_INDX(RingBuffer* reader,
     } break;
   }
 
+  auto viz_query = register_file_->Get<reg::PA_SC_VIZ_QUERY>();
+  if (viz_query.viz_query_ena && viz_query.kill_pix_post_hi_z) {
+    // TODO(Triang3l || JoelLinn) Handle this properly in the render backends.
```

Comment:
Small nitpick again since other changes are requested — we have : after TODO(name).